src/esp/assets/managers/SceneAttributesManager.cpp

```diff
@@ -268,11 +371,70 @@ SceneAttributesManager::createFileBasedAttributesTemplate(
       jsonConfig, "origin",
       std::bind(&PhysicsSceneAttributes::setOrigin, sceneAttributes, _1));
 
-  // semantic asset handle for scene
-  io::jsonIntoSetter<std::string>(
-      jsonConfig, "semantic asset handle",
-      std::bind(&PhysicsSceneAttributes::setSemanticAssetHandle,
-                sceneAttributes, _1));
+  // populate specified semantic file name if specified in json - defaults
+  // are overridden only if specified in json.
+  std::string semanticFName = "";
+  std::string navmeshFName = "";
+  std::string houseFName = "";
+  std::string lightSetup = "";
+  if (io::jsonIntoVal<std::string>(jsonConfig, "semantic mesh",
+                                   semanticFName)) {
+    semanticFName =
+        Cr::Utility::Directory::join(sceneLocFileDir, semanticFName);
+  }
+  if (semanticFName != "") {
+    // if "semantic mesh" is specified in scene json to non-empty value, set
+    // value (override default).
+    sceneAttributes->setSemanticAssetHandle(semanticFName);
+  }
+
+  if (io::jsonIntoVal<std::string>(jsonConfig, "nav mesh", navmeshFName)) {
+    navmeshFName = Cr::Utility::Directory::join(sceneLocFileDir, navmeshFName);
+  }
+  if (navmeshFName != "") {
+    // if "nav mesh" is specified in scene json to non-empty value, set value
+    // (override default).
+    sceneAttributes->setNavmeshAssetHandle(navmeshFName);
+  }
+
+  if (io::jsonIntoVal<std::string>(jsonConfig, "house filename", houseFName)) {
+    houseFName = Cr::Utility::Directory::join(sceneLocFileDir, houseFName);
+  }
+  if (houseFName != "") {
+    // if "house filename" is specified in scene json to non-empty value, set
+    // value (override default).
+    sceneAttributes->setHouseFilename(houseFName);
+  }
+
+  if (io::jsonIntoVal<std::string>(jsonConfig, "lighting setup", lightSetup)) {
+    // if lighting is specified in scene json to non-empty value, set value
+    // (override default).
+    if (lightSetup != "") {
```

Comment:
Remove this if: "" is the default light key DEFAULT_LIGHTING_KEY, so we should accept this.